window.py

```diff
@@ -39,35 +41,30 @@ def assertSet(value: any) -> bool:
         if directory == "":
             print("Nothing selected")
             continue
-        clean_dir = directory[(directory.rfind("/")+1):]
+        clean_dir = directory[(directory.rfind("/")+1):] # TODO: use appropriate function from os.path
```

Comment:
I am exactly sure what the intention was here, if it's getting the dir then probably use `os.path.dirname(path)` if it was to remove trailing slash maybe the same works as well.

Reply:
The intention was to have a simple filename/dirname of the whatever is currently selected to display in UI instead of the entire path

Reply:
Oh I see, I didn't read that right the first time. We are cutting the left hand side of the string. In this case we would use `os.path.basename(path)`.